### PR TITLE
Remove `sorries` from `entropy_pfr` 

### PR DESCRIPTION
**Pull request by Paul-Lez**
This PR makes a couple of modifications to `entropy_pfr` in order to get rid of the last sorries in `entropic_PFR_conjecture'`. 
I also ended up breaking up a few long lines in the files I was looking at (all the changes in `entropy_basic` are of this form) in order to make the code a bit more readable (I initially hadn't planned on doing this so I'm happy to revert these changes/do them in another PR if necessary)

### CODE REVIEW

**Comment by Paul-Lez** [comment]
This builds locally so hopefully it also builds here!

**Comment by Paul-Lez** [comment]
Apparently there are conflicts, I'll deal with those now!

**Comment by Paul-Lez** [comment]
Another build error - I'll fix this now

File: PFR/Entropy/Basic.lean [hunk 1]
```diff
@@ -291,9 +291,9 @@ lemma entropy_comm
   exact (entropy_comp_of_injective μ (hX.prod_mk hY) Prod.swap Prod.swap_injective).symm
 
 /-- $H[(X,Y),Z] = H[X,(Y,Z)]$. -/
-lemma entropy_assoc [MeasurableSingletonClass S] [MeasurableSingletonClass T] [MeasurableSingletonClass U]
-    (hX : Measurable X) (hY : Measurable Y) (hZ : Measurable Z) (μ : Measure Ω) :
-    H[⟨ X, ⟨ Y, Z ⟩ ⟩; μ] = H[⟨ ⟨X, Y⟩ , Z ⟩ ; μ] := by
+lemma entropy_assoc [MeasurableSingletonClass S] [MeasurableSingletonClass T]
+  [MeasurableSingletonClass U] (hX : Measurable X) (hY : Measurable Y) (hZ : Measurable Z)
+  (μ : Measure Ω) : H[⟨ X, ⟨ Y, Z ⟩ ⟩; μ] = H[⟨ ⟨X, Y⟩ , Z ⟩ ; μ] := by
```
Comment (sgouezel) [inline PFR/Entropy/Basic.lean:296]: This change is wrong, you should have indentation by 4 spaces.

**Comment by teorth** [comment]
Yay! It passed all the checks.

**Pull request merged**